extensions/amp-list/0.1/amp-list.js

```diff
@@ -551,6 +587,14 @@ export class AmpList extends AMP.BaseElement {
           });
         }
         removeChildren(dev().assertElement(this.container_));
+      };
+
+      if (!this.loadMoreEnabled_ && this.enableManagedResizing_) {
+        this.lockHeightAndMutate_(reset);
+        return;
+      }
+      this.measureElement(() => {
```

Comment:
Did we mean to change this from a mutate to a measure?

Reply:
No, thanks! 😅